.github/workflows/recipe.yaml

```diff
@@ -19,8 +19,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.4]
-        sylius: [1.8, 1.9]
+        php: ['7.4' ,'8.0']
+        sylius: ["~1.8.0", "~1.9.0", "~1.10.0"]
+        exclude:
+          - php: 8.0
+            sylius: "~1.8.0"
+          - php: 8.0
+            sylius: "~1.9.0"
```

Comment:
Do you think it will works with that ?

```
          - php: 8.0
            sylius: ["~1.8.0", "~1.9.0"]
```

Reply:
not sure but let's have a try

Reply:
It does not ^^

I lett you revert it and after we will be able to merge \o/